zookeeper/scripts/zookeeper_run.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
 
-# volume for saving Kafka server logs
+# volume for saving Zookeeper server logs
 export ZOOKEEPER_VOLUME="/tmp/zookeeper/"
-# base name for Kafka server data dir and application logs
+# base name for Zookeeper server data dir and application logs
 export ZOOKEEPER_DATA_BASE_NAME="data"
 export ZOOKEEPER_LOG_BASE_NAME="logs"
+# Disable JMX until we need it
+export JMXDISABLE=true
 
-export ZOOKEEPER_ID=$(hostname | awk -F'-' '{print $2}')
+# TODO: Zookeeper cluster support. 
```

Comment:
I think this `TODO` would be better captured in an issue.

Reply:
I raised #22 to cover this and removed the TODO comment.